src/test/java/io/supertokens/test/UsersAPITest2_5.java

```diff
@@ -0,0 +1,191 @@
+/*
+ *    Copyright (c) 2020, VRAI Labs and/or its affiliates. All rights reserved.
+ *
+ *    This software is licensed under the Apache License, Version 2.0 (the
+ *    "License") as published by the Apache Software Foundation.
+ *
+ *    You may not use this file except in compliance with the License. You may
+ *    obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *    License for the specific language governing permissions and limitations
+ *    under the License.
+ */
+
+package io.supertokens.test;
+
+import io.supertokens.ProcessState;
+import io.supertokens.pluginInterface.STORAGE_TYPE;
+import io.supertokens.storageLayer.StorageLayer;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TestRule;
+
+import java.util.HashMap;
+
+import static org.junit.Assert.*;
+
+/*
+ * TODO:
+ *  - Check for bad input (missing fields)
+ *  - Check good input works (add 5 users)
+ *    - no params passed should return 5 users
+ *    - only limit passed (limit: 2. users are returned in ASC order based on timeJoined)
+ *    - limit and timeJoinedOrder passed (limit: 2, timeJoinedOrder: DESC. users are returned in DESC order based on timeJoined)
+ *    - limit = 5, nextPaginationToken should not be present in the response
+ *    - remove all users from db, response should not have any user and nextPaginationToken should not be present
+ *    - set limit 2, call the API, from the response use nextPaginationToken to call API again. from the response use nextPaginationToken to call API again. in the response of the final (3rd) API call, only one item is returned and nextPaginationToken is not present in the response
+ * */
```

Comment:
The test cases here are already covered in the EmailPasswordTest_2 right? So here it's just checking that the API's responses are proper for different, but simple cases.